clang/test/SemaOpenCL/builtins-amdgcn-error-unsupported-on-gfx12.cl

```diff
@@ -0,0 +1,105 @@
+// REQUIRES: amdgpu-registered-target
```

Comment:
Maybe just add these to `test/CodeGenOpenCL/builtins-amdgcn-gfx12-err.cl` instead of a new file?